packages/gatsby/package.json

```diff
@@ -46,13 +46,13 @@
     "domready": "^1.0.8",
     "dotenv": "^4.0.0",
     "eslint": "^4.19.1",
-    "eslint-config-react-app": "^2.1.0",
+    "eslint-config-react-app": "3.0.0-next.66cc7a90",
     "eslint-loader": "^2.0.0",
     "eslint-plugin-flowtype": "^2.46.1",
     "eslint-plugin-graphql": "^2.0.0",
     "eslint-plugin-import": "^2.9.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.7.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
```

Comment:
👍 this is the fix, isn't it?

Reply:
As I have mentioned before, eslint-config-react-app@2.1.0 refers to a deprecated rule (`jsx-a11y/href-no-hash`) from eslint-plugin-jsx-a11y@5, which has been replaced (with `jsx-a11y/anchor-is-valid`) in eslint-plugin-jsx-a11y@6.

Reply:
👍